panel/pane/base.py

```diff
@@ -1,13 +1,13 @@
 """
-Defines the PaneBase class defining the API for panes which convert
-objects to a visual representation expressed as a bokeh model.
+Defines base classes for Pane components which allow wrapping a Python
+object transforming it into a Bokeh model that can be rendered.
```

Comment:
Would probably update with:
`...object and transforming...`
`...that can be rendered in the browser.`